build/buildBuildImages.sh

```diff
@@ -410,7 +410,8 @@ function buildCliImage() {
 
 	if [ -z "$debianFlavor" ] || [ $debianFlavor == "stretch" ] ; then
 		debianFlavor="stretch"
-		builtImageName="$builtImageName:debian-$debianFlavor"
+		#Change buildImage name to fix validation pipeline
+		builtImageName="$builtImageName-$debianFlavor:debian-$debianFlavor"
```

Comment:
Could you please provide more context about the double -$debianFlavor info? I see that we do it in the `else` condition but this seems redundant. Do you have more info?

Reply:
Thanks @pauld-msft for the explanation! Yes, I agree I would rather go on the long term with a consistent approach, specifically `oryx/build:cli-debian-buster-version`. It feels like whatever we can simplify in terms of images is a win, so it would go this way. If we rename this, we could even go when step further and rename the CLI image to "skeleton" as we talked about in the past, to remove even further the ambiguity about what is included in there.

Now in terms of short term work, we're not going to do this in this PR. I'm not sure however if the move to `cli-stretch` puts us closer to the long term goal, or if it will make things more complicated later. If it's neutral, then I'm fine with it, if we expect this would just create more work for us later, then I would rather stick with the old format. @pauld-msft @snehapar9 Any thoughts?

Reply:
Chatted with @snehapar9 about this offline -- I'm in the boat that we should move to the single repo approach of `oryx/cli:debian-{flavor}-{tag}` since (1) we already have the `oryx/cli` repository registered with MCR and (2) we already follow that pattern with our `oryx/build` images and would make sense to be consistent here.

I don't think this would be a large amount of work, and would probably warrant being done in its own separate PR as to not "bloat" this PR with changes "outside of the dependency changes being made". That being said, it's easy for me to say that as the person not making the changes (:smile:) so I'd make sure we could timebox the effort to make the changes, and if it's not anything too crazy, we could open that PR soon, get it merged, and incorporate the new tag structure in this PR.

Any thoughts?

Reply:
I'm on board with the single cli repo approach of `oryx/cli:debian-{flavor}-{tag}` and agree that it would be good to have a separate PR that merges into this one. I agree that it shouldn't be a _ton_ of work, as the areas that need to be changed are mostly highlighted in this pr AFAIK. 

It would be a step in the right direction towards consistency, and I think it would be good to do here instead of waiting to do the slightly larger conversion to `oryx/build:cli-debian-{flavor}-{tag}` at some point in the future, since I don't really see us prioritizing that work soon.

Reply:
+1. I'm onboard with just using the single cli repo approach for all the debian flavors as well. Would be easier to make this change along with this PR now than re-doing this at some point in the future and having to undo changes in Oryx.yml file.

I do not expect other files outside of the files included in this PR to change.

Reply:
> Chatted with @snehapar9 about this offline -- I'm in the boat that we should move to the single repo approach of `oryx/cli:debian-{flavor}-{tag}` since (1) we already have the `oryx/cli` repository registered with MCR and (2) we already follow that pattern with our `oryx/build` images and would make sense to be consistent here.
> 
> I don't think this would be a large amount of work, and would probably warrant being done in its own separate PR as to not "bloat" this PR with changes "outside of the dependency changes being made". That being said, it's easy for me to say that as the person not making the changes (😄) so I'd make sure we could timebox the effort to make the changes, and if it's not anything too crazy, we could open that PR soon, get it merged, and incorporate the new tag structure in this PR.
> 
> Any thoughts?
It would actually be easier adding these cli repo changes in the current PR. Opening a new PR adds other redundant changes and conflicts.